pkg/cmd/generate/cmd.go

```diff
@@ -165,11 +177,24 @@ func extractDepPRs(prs []github.PullRequest) (depPRs, otherPRs []github.PullRequ
 	depPRs = make([]github.PullRequest, 0)
 	otherPRs = make([]github.PullRequest, 0)
 	for i := range prs {
-		pr := &prs[i]
-		if Contains(pr.Labels, "dependencies") {
-			depPRs = append(depPRs, prs[i])
-		} else {
-			otherPRs = append(otherPRs, prs[i])
+		pr := prs[i]
+		messageLines := strings.Split(strings.ReplaceAll(pr.RelatedCommit.Message, "\r\n", "\n"), "\n")
+		for i := range messageLines {
+			messageLine := messageLines[i]
+			if dependabotPrefix.Match([]byte(messageLine)) {
+				depPRs = append(depPRs, github.PullRequest{
```

Comment:
Possibly simplify this by adding a Clone method to the PulRequest object and just set the Title on the Clone?

Reply:
Good point, thanks.